tests/integration_tests/functional/test_balloon.py

```diff
@@ -615,7 +615,8 @@ def _test_balloon_snapshot(context):
     # Get the firecracker from snapshot pid, and open an ssh connection.
     firecracker_pid = microvm.jailer_clone_pid
 
-    # Get the stats right after we take a snapshot.
+    # Wait out the polling interval, then get the updated stats.
+    time.sleep(1)
```

Comment:
Ah I see. Maybe define `STATS_POLLING_INTERVAL_S` at the top of the file and use that everywhere?

Reply:
That was my initial idea. However the test file uses different polling intervals in some cases, and having separate constants for all of them looks a bit too much to me. Moreover, some of the delays are not related to the stats polling.

Reply:
hmm maybe a test-local one? My main concern is we will have to change the `1` in two places vs one, if we ever change it.

Reply:
Yes, that is very true.

Reply:
Added ```STATS_POLLING_INTERVAL_S``` and used it where it matters, left others intact.